_posts/2017/09/2017-09-02-merger.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+authors: ["Rayna Harris", "Tracy Teal"]
+title: "Joint future for Software Carpentry and Data Carpentry"
+date: 2017-09-02
+time: "00:00:00"
+category: [ "Steering Committee", "Community", "Data Carpentry"]
+---
+
+*“If you want to go fast, go alone. If you want to go far, go together.”*
+
+Software Carpentry and Data Carpentry are currently independent organizations with their own fiscal sponsorship, Steering Committees, governance model, and bank accounts. However, as is perhaps no surprise the organizations' operations have evolved to share memberships, infrastructure for workshop coordination, an instructor training program, and even some staff members. This 'separate but collaborative' organizational structure has allowed us to build a shared community of instructors with more than 1000 certified instructors and 47 current Member Organizations around the world.
```

Comment:
Is it obvious for most people that these orgs are really sister orgs? I've been involved for a long time, and I know the "origin story", so to speak, but just wondering what you guys think. If you don't think the shared origins are obvious to all, I suggest putting in a sentence or two at the beginning that explains that.